README.md

```diff
@@ -64,6 +64,37 @@ The build process also creates ```.stamps``` and ```.tmp-output```. The first co
 stamp files to keep track of which parts have been built, while the second contains
 intermediate build output (e.g., kernel images and ISO files).
 
+# Building Windows images
+
+You need to first get the ISO file for the Windows version you would like to install.
+You can download these images from MSDN. The hash and the name of the ISO is
+specified in the  ``images.json`` file. Place these files in the ``iso`` folder.
```

Comment:
Place "these" files in the ``iso`` folder

Which files? The ISO and a file containing the hash? Just the ISO? "These" makes it sound like multiple files, so I'm a bit confused. Please clarify.